<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>name_set_form</class>
 <widget class="QWidget" name="name_set_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>맑은 고딕</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Setting_Bt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>700</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>설정</string>
   </property>
  </widget>
  <widget class="QPushButton" name="can_Bt">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>700</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>371</width>
     <height>657</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="num_arr">
      <item>
       <widget class="QLabel" name="num_labal">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1번 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="num_labal_2">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>2번 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="num_labal_3">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1번 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="num_labal_4">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1번 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="num_labal_5">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1번 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="num_labal_6">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1번 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="num_labal_7">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>7번 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="num_labal_8">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1번 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="num_labal_9">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1번 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="num_labal_10">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>10번 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="num_labal_11">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1번 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="num_labal_12">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1번 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="num_labal_13">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1번 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="num_labal_14">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1번 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="num_labal_15">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1번 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="num_labal_16">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1번 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="num_labal_17">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1번 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="num_labal_18">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1번 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="num_labal_19">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1번 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="num_labal_20">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1번 : </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="name_arr">
      <item>
       <widget class="QLineEdit" name="name_text"/>
      </item>
      <item>
       <widget class="QLineEdit" name="name_text_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="name_text_3"/>
      </item>
      <item>
       <widget class="QLineEdit" name="name_text_4"/>
      </item>
      <item>
       <widget class="QLineEdit" name="name_text_5"/>
      </item>
      <item>
       <widget class="QLineEdit" name="name_text_6"/>
      </item>
      <item>
       <widget class="QLineEdit" name="name_text_7"/>
      </item>
      <item>
       <widget class="QLineEdit" name="name_text_8"/>
      </item>
      <item>
       <widget class="QLineEdit" name="name_text_9"/>
      </item>
      <item>
       <widget class="QLineEdit" name="name_text_10"/>
      </item>
      <item>
       <widget class="QLineEdit" name="name_text_11"/>
      </item>
      <item>
       <widget class="QLineEdit" name="name_text_12"/>
      </item>
      <item>
       <widget class="QLineEdit" name="name_text_13"/>
      </item>
      <item>
       <widget class="QLineEdit" name="name_text_14"/>
      </item>
      <item>
       <widget class="QLineEdit" name="name_text_15"/>
      </item>
      <item>
       <widget class="QLineEdit" name="name_text_16"/>
      </item>
      <item>
       <widget class="QLineEdit" name="name_text_17"/>
      </item>
      <item>
       <widget class="QLineEdit" name="name_text_18"/>
      </item>
      <item>
       <widget class="QLineEdit" name="name_text_19"/>
      </item>
      <item>
       <widget class="QLineEdit" name="name_text_20"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
